test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssertTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Mvc.Xml.Test
```

Comment:
no `.Test` in test namespaces; always use namespace of the test subject - `Microsoft.AspNet.Mvc.Xml` in this case